server/service/logging.go

```diff
@@ -15,3 +16,11 @@ type loggingMiddleware struct {
 func NewLoggingService(svc kolide.Service, logger kitlog.Logger) kolide.Service {
 	return loggingMiddleware{Service: svc, logger: logger}
 }
+
+// loggerForError returns a logger with a log level dependant on error
+func (lm *loggingMiddleware) loggerForError(err error) kitlog.Logger {
+	if err != nil {
+		return level.Error(lm.logger)
```

Comment:
I prefer Info vs Error. 

https://dave.cheney.net/2015/11/05/lets-talk-about-logging

Reply:
well there was an error. I guess it depends how that error is in turn handled but the logger doesn't have any visibility of that. Filtering in something like kibana is a lot nicer when you can say: 'level = error' rather than something more complex like 'err != "nill" - it feels more consistent in a multi tenanted environment with different systems feeding into the same log collector

For my use case I don't much mind -- I just need the mass of useless log lines triggered by every single asset doing anything with kolide-fleet gone.  If it's a blocker Im totally fine with this been Info.

Reply:
also I don't see this as *handling* the error in the classic sense. These log events are deffered after the handler call has completed and really just provide observability. The actual error may or may not be handled futher up in the call stack (I have no idea if some other part of middleware) swallows the error or hadles it in some other way.

Reply:
I do agree with some of what you're saying. It's ultimately up to @zwass to make a choice here.

For context, you don't really care about the logging here, so maybe that's an argument for dropping even errors to debug. They're handled by the http response, which _should_ (not necessarily happening in this code) be also logging the response to the client. 
It's important to care for an admin what got sent to the client ( status + reason) and perhaps alert on those. In the middleware it's more important to the developer.